lib/main.dart

```diff
@@ -41,16 +40,16 @@ void main() async {
   ]);
 
   // Pull firebase data from local emulators in dev
-  if (kDebugMode) {
-    try {
-      FirebaseFirestore.instance.useFirestoreEmulator('localhost', 8080);
-      FirebaseStorage.instance.useStorageEmulator('localhost', 9199);
-      await FirebaseAuth.instance.useAuthEmulator('localhost', 9099);
-    } catch (e) {
-      // ignore: avoid_print
-      print(e);
-    }
-  }
+  // if (kDebugMode) {
+  //   try {
+  //     FirebaseFirestore.instance.useFirestoreEmulator('localhost', 8080);
+  //     FirebaseStorage.instance.useStorageEmulator('localhost', 9199);
+  //     await FirebaseAuth.instance.useAuthEmulator('localhost', 9099);
+  //   } catch (e) {
+  //     // ignore: avoid_print
+  //     print(e);
+  //   }
+  // }
```

Comment:
@cryptixcoder is it intentional to leave this commented out rather than leveraging the `kDebugMode` flag?

Reply:
@themightychris It was intentional when I was ensuring that the real world data that we had in the production firestore looked as expected. But I will uncomment that code so that we have the debug version of the app use the emulator going forward.